### ISSUE TITLE
maxLengthEnforced removed in TextFormField

### ISSUE THREAD

**Issue by aideric** [peiffer-innovations/automated_testing_framework#9]
In the latest Flutter 3.0. maxLengthEnforced maybe completley removed by flutter. 
```
   ../../.pub-cache/hosted/pub.dartlang.org/automated_testing_framework-3.3.1+1/lib/src/widgets/testable_text_form_field.dart:181:11: Error: No named parameter with the name 'maxLengthEnforced'.
              maxLengthEnforced: widget.maxLengthEnforced,
              ^^^^^^^^^^^^^^^^^
```

**Comment by aideric** [comment]
Hi, I found that maxLengthEnforced is replaced by maxLengthEnforcement.
[Deprecated API removed after v2.10](https://docs.flutter.dev/release/breaking-changes/2-10-deprecations)

**Comment by jpeiffer** [comment]
Thanks.  This has been resolved today with 4.0.0.
